docs/CONTRIBUTING.md

```diff
@@ -184,6 +185,13 @@ any pull request with:
 
 When deprecating functionality in Vector, see [DEPRECATION.md](DEPRECATION.md).
 
+### Dependencies
+
+When adding, modifying, or removing a dependency in Vector you may find that you need to update the
+inventory of third-party licenses maintained in `LICENSE-3rdparty.csv`. This file is generated using
+[rust-license-tool](https://github.com/DataDog/rust-license-tool.git) and can be updated using
+`cargo vdev build licenses`.
```

Comment:
Is it worth calling out `vdev check licenses` too, or just leave that since it's a more CI-focused subcommand?

Reply:
Yeah, I figured I'd leave that out since it exists primarily for CI.